src/middlewared/middlewared/plugins/chart_releases_linux/normalize_schema.py

```diff
@@ -159,3 +160,24 @@ async def normalise_certificate_authorities(self, attr, value, complete_config,
         )
 
         return value
+
+    @private
+    async def normalise_acl(self, attr, value, complete_config, context):
+        assert isinstance(attr, Dict) is True
+
+        if not value:
+            return value
+
+        action_dict = next((d for d in context['actions'] if d['method'] == 'apply_acls'), None)
```

Comment:
Just a preference but I like to be explicit with checks. i.e.

```suggestion
        if (action_dict := next((d for d in context['actions'] if d['method'] == 'apply_acls'), None)) is not None:
```